src/VisualStudio/CSharp/Test/CSharpVisualStudioTest.csproj

```diff
@@ -180,6 +180,22 @@
     <Reference Include="System.Xml.Linq" />
   </ItemGroup>
   <ItemGroup>
+      <VSIXSourceItem Include="$(NuGetPackageRoot)\SQLitePCLRaw.bundle_green\1.1.2\lib\net45\SQLitePCLRaw.batteries_green.dll" />
```

Comment:
This shouldn't be necessary for tests - they aren't VSIXs.  You just need it in the output directory, which I think happens just by referencing the assembly.